skyportal/handlers/api/instrument.py

```diff
@@ -692,6 +710,23 @@ def delete(self, instrument_id):
                         has no filters (e.g., because it is a spectrograph),
                         leave blank or pass the empty list.
                       default: []
+                    sensitivity_data:
+                      type: object
+                      properties:
+                        filter_name:
+                          type: object
+                          enum: {list(ALLOWED_BANDPASSES)}
+                          properties:
+                            limiting_magnitude:
+                              type: float
+                            magsys:
+                              type: string
+                            exposure_time:
+                              type: float
+                              description: >-
+                                Exposure time in seconds.
```

Comment:
Is there a difference between using `>-` and `|` after the description?

Reply:
I am switching over to |